src/transformers/core/createFilter.ts

```diff
@@ -20,4 +16,16 @@ export function createFilter(filter: any): Filter {
     matcher: matcher as Matcher,
     value
   };
+}
+
+function createNumberRangeValue(value: any): NumberRangeValue {
+  const numberRangeValue: NumberRangeValue = {};
+  Object.keys(value).forEach(limitMatcher => {
+    if (limitMatcher === Matcher.GreaterThan || limitMatcher === Matcher.GreaterThanOrEqualTo) {
```

Comment:
can we use the Upper/Lower NumberRangeMatchers here?

Reply:
the NumberRangeMatchers are types so they can't be use here in the conditional logics as values